pkgs/development/python-modules/django-scim2/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonRelaxDepsHook
+
+# propagates
+, django
+, mock
+, dateutil
+, scim2-filter-parser
+, gssapi
+, ldap
+, sssd
+
+# tests
+, djangorestframework
+, pytest-django
+, pytest-xdist
+, pytest-mock
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "django-scim2";
+  version = "0.17.3";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "15five";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-5zdGPpjooiFoj+2OoglXhhKsPFB/KOHvrZWZd+1nZqU=";
+  };
+
+  buildInputs = [ sssd ];
+  propagatedBuildInputs = [
+    django
+    mock
+    dateutil
+    scim2-filter-parser
+    gssapi
+    ldap
+    sssd
+  ];
+
+  nativeBuildInputs = [ pythonRelaxDepsHook ];
+  pythonRelaxDeps = [
+    "django"
+  ];
+
```

Comment:
What is with tests and pythonImportsCheck?

Reply:
Added.